### DISCUSSION TITLE
Use installer on Windows and then paste over linux files?

### DISCUSSION THREAD

**Discussion by WoundedBum** [LostArtefacts/TRX#1175] [Q&A]
Hi,

I'm just wondering - I have installed the game and configured it extensively as I'd like on windows including the music etc, then pasted the files over to my steamdeck. That didn't really work, I kept having to bugger about with dlls etc. But then I just downloaded the Linux zip and overwrote all the files (I hadn't created a config file on the Deck itself) while keeping all downloaded additional files etc. It seems to be working just fine, but I can't test extensively the later parts of the game. In theory am I likely to run into any issues? 

Thanks

**Comment by walkawayy** [comment]
If it's working that way then go for it! You can use the console or cheats to skip around the levels and test if you'd like.
